hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/sigs/order/SigRequirementsTest.java

```diff
@@ -6485,12 +6485,4 @@ public static List<Key> sanityRestored(List<? extends HederaKey> jKeys) {
                         })
                 .collect(toList());
     }
-
-    public static Key sanityRestored(HederaKey jKey) {
-        try {
-            return JKey.mapJKey((JKey) jKey);
-        } catch (Exception ignore) {
-            throw new AssertionError("All keys should be mappable!");
-        }
-    }
 }
```

Comment:
I don't see any tests ported from this class (`SigRequirementsTest`) to test the new code. We want to make sure we have the same functionality in both the new and old code.